packages/Manager/src/main/resources/application.yml

```diff
@@ -21,6 +23,21 @@ spring:
         spring.json.value.type.method: nz.govt.eop.messages.KafkaMessageTypes.determineTypeFromTopicName
         spring.json.trusted.packages: nz.govt.eop.*
 
+    streams:
+      properties:
+        num.stream.threads: 8
+        max.poll.records: 100
```

Comment:
This was also a memory improvement change, by default this was 500. This is how many records are pulled from Kafka in a batch.